lib/actions/addLinkToProjectAction.js

```diff
@@ -0,0 +1,28 @@
+class AddLinkToProjectAction {
+
+  // validate(){}
```

Comment:
Let's delete this. We have history to review commented out code and we don't really know what this might look like.